style.css

```
@@ -129,6 +161,18 @@ a {
     text-decoration: underline;
 }
 
+#showSerialHelp {
+    background: none;
+    border: none;
+    color: #fff;
+    cursor: pointer;
+    text-decoration: underline;
+    padding: 0;
```

Stylelint (17.14.1)

[error] Duplicate property "padding" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)